packages/core/src/DelegateMixin.js

```diff
@@ -1,203 +1,179 @@
 /* eslint-disable class-methods-use-this */
-
 import { dedupeMixin } from '@open-wc/dedupe-mixin';
 
 /**
- * # DelegateMixin
```

Comment:
These docs/comments are now in the manually written type definition file so you get it as a popup on the typehint